.github/workflows/ci.yml

```diff
@@ -181,8 +181,8 @@ jobs:
           name: ubuntu-binaries
           path: build/bin
   ubuntu_20_04_clang:
```

Comment:
The name of the job was not updated

Reply:
Good catch, updated in ccb13b396bc13ea6786ff612b5ca771357795ca6